style.css

```diff
@@ -31,70 +36,327 @@ header {
-  font-family: 'poppins', sans-serif;
-  padding: 70px 20px 0 20px;
+  padding: 130px 20px 0 20px;
   line-height: 52px;
 }
 
-.sub {
+.sub-heading {
   color: #344563;
   font-size: 16px;
   font-weight: 400;
-  font-family: 'poppins', sans-serif;
-  padding: 20px 24px;
+  padding: 10px 24px;
   line-height: 24px;
-  margin-top: -40px;
+  margin-top: 5px;
   margin-bottom: -20px;
 }
 
 /* Designig social */
+.connect-top {
+  margin: 20px 0 20px 25px;
+  font-size: 16px;
+  font-weight: 500;
+  line-height: 24px;
+  letter-spacing: 0;
+  text-align: left;
+}
 
-.icon {
+.connect-top > * {
+  color: #7f8cff;
+}
+
+.social-icon {
   display: flex;
   flex-direction: row;
-  list-style: none;
-  flex: 1 1 10px;
-  align-content: space-between;
-  padding: 10px;
-  height: 20px;
+  padding-left: 15px;
+  height: 60px;
   flex-wrap: nowrap;
-  margin-top: -20px;
+  margin-top: -15px;
 }
 
-.icon li {
+.social-icon > * {
   padding: 10px;
 }
 
-.contact {
-  padding: 0 0 0 20px;
-  font-family: 'Poppins', sans-serif;
-  font-size: 16px;
+li {
+  list-style: none;
+}
+
+a {
+  text-decoration: none;
+}
+
+/* signig worection */
+.workcards {
+  display: grid;
+  row-gap: 4em;
+  padding: 100px 25px;
+}
+
+.card {
+  background-color: #fff;
+  padding: 20px;
+  border-radius: 8px;
+  min-height: 65vh;
+  display: flex;
+  flex-direction: column;
+  border: 1px solid #dfe1e6;
+}
+
+.project-picture img {
+  width: 100%;
+}
+
+.project-title {
+  font-size: 32px;
+  font-weight: 700;
+  line-height: 44px;
+  letter-spacing: 0;
+  text-align: left;
+  color: #344563;
+  margin: 0 0 20px 0;
+}
+
+.frame-container {
+  display: flex;
+  flex-direction: row;
+  text-decoration: none;
+  margin-top: -12px;
+  font-family: Poppins, sans-serif;
+  font-size: 13px;
+  font-weight: 600;
+  line-height: 16px;
+}
+
+.frame {
+  padding: 0 12px 0 0;
+}
+
+.client {
+  color: #344563;
+}
+
+.role,
+.year {
+  color: #7a869a;
+}
+
+.card-text {
+  font-size: 15px;
+  font-weight: 400;
+  line-height: 24px;
+  letter-spacing: 0;
+  text-align: left;
+  color: #344563;
+  margin-top: 15px;
+}
+
+.card-language-container {
+  display: flex;
+  flex-direction: row;
+}
+
+.card-language {
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 16px;
+  letter-spacing: 0.03em;
+  text-align: left;
+  background-color: #ebebff;
+  color: #6070ff;
+  border-radius: 8px;
+  padding: 4px 12px;
+  margin: 10px;
+  margin-left: 0;
+}
+
+.card-btn {
+  text-align: left;
+  background-color: white;
+  color: #396df2;
+  width: fit-content;
+  border-radius: 8px;
+  font-size: 17px;
   font-weight: 500;
   line-height: 24px;
-  letter-spacing: 0.1em;
+  letter-spacing: 0.03em;
+  border: 1px solid #396df2;
+  padding: 12px;
+  margin-top: 15px;
+}
+
+.card-btn:hover {
+  background-color: #6070ff;
+  color: white;
+}
+
+.card-btn:active {
+  background-color: #2230d2;
+  color: white;
+}
+
+.about-contact-container {
+  background-color: #fff;
+  border-radius: 0 198px 0 0;
```

Comment:
- [ ] Yes, you have added the border-radius but it is not well adjusted as per the original design, your border is more curved and large than the original one,  please refix it taking reference to the figma template

<img width="549" alt="1" src="https://github.com/ali-techworld/Portfolio-Setup/assets/77843050/eb9ef00b-8d7e-422e-b3e7-cbae931c31fd">